src/jdk.accessibility/windows/classes/com/sun/java/accessibility/internal/AccessBridge.java

```diff
@@ -1593,6 +1607,14 @@ public Rectangle call() throws Exception {
                             if (p != null) {
                                 r.x = p.x;
                                 r.y = p.y;
+
+                                AffineTransform at = getTransformFromContext(ac);
+                                if (at != null) {
+                                    r.x = (int) Math.floor(r.x * at.getScaleX());
+                                    r.y = (int) Math.floor(r.y * at.getScaleY());
+                                    r.width = (int) Math.ceil(r.width * at.getScaleX());
+                                    r.height = (int) Math.ceil(r.height * at.getScaleY());
+                                }
```

Comment:
This logic will not work on the multiscreen configs when the monitors have different DPI, the logic should be similar to the SunGraphicsEnvironment.toDeviceSpace().
and probably the round logic should use Region.clipRound as well.

Reply:
If this data will be used in native then we can try to use device->ScaleDownAbsX/Y, etc methods.